### PR TITLE
Adds CRUD example to Database-page (could fix #3617)

### LINKED ISSUE #3617: Database-article should show how to query the database
Hi!

The article about a custom table explains well how to create a table in V9, but it does not show how to query the table and the best practice around this.

***This is the page with issues: https://github.com/umbraco/UmbracoDocs/edit/main/Extending/Database/index.md***

### PR DESCRIPTION
**Pull request by jannikanker**

### CODE REVIEW

File: Extending/Database/index.md [hunk 1]
```diff
@@ -298,3 +298,49 @@ When storing data in custom database tables, this is by default not manageable b
...
 
 Figuring out how to manage data across multiple environments can be different between individual sites and there is not one solution that fits all. Some sites may have automated database synchronization set up to ensure specific tables in multiple databases are always kept in sync. Other sites may be better off with scripts moving data around manually on demand.
+
+## Working with data in Custom Database Tables
+
+To create, read, update or delete data from your custom database tables, you can use the `IScopeProvider` to get access to the database operations.
+
+The following example creates an `UmbracoApiController` to be able to fetch and insert blog comments.
+
+Note that this example does **NOT** use the aforementioned `BlogCommentSchema` class but rather a seperate (yet duplicate) class - as recommended above - which is not part of the example. Also be aware that error handling, data validation and such has been omitted for brevity.
```
Comment (sofietoft) [inline Extending/Database/index.md:308]:
```suggestion
:::note
This example does not use the aforementioned `BlogCommentSchema` class but rather a separate (yet duplicate) class that is not part of the example. Also, be aware that things like error handling and data validation have been omitted for brevity.
:::
```

**Review by sofietoft — COMMENTED** [review]
Thanks a bunch for the PR @jannikanker ! 💪 

I've added a suggestion to shorten a sentence and structure it a little different.
Let me know what you think!

Also, I'll have it by our developers for a checkup 💪

**Comment by jannikanker** [comment]
> Thanks a bunch for the PR @jannikanker ! 💪
> 
> I've added a suggestion to shorten a sentence and structure it a little different. Let me know what you think!
> 
> Also, I'll have it by our developers for a checkup 💪

No problem, I remembered that I myself had been missing this very example not too long ago :-D

Your edit makes all the sense [tm] ;-)

File: Extending/Database/index.md [hunk 2]
```diff
@@ -298,3 +298,51 @@ When storing data in custom database tables, this is by default not manageable b
...
+        public BlogCommentsApiController(IScopeProvider scopeProvider)
+        {
+            _scopeProvider = scopeProvider;
+        }
+
+        [HttpGet]
+        public IEnumerable<BlogComment> GetComments(int umbracoNodeId)
+        {
+            using var scope = _scopeProvider.CreateScope();
+            var queryResults = scope.Database.Fetch<BlogComment>("SELECT * FROM BlogComments WHERE BlogPostUmbracoId = @0", umbracoNodeId);
```
Comment (Zeegaan) [inline Extending/Database/index.md:333]:
```suggestion
            var queryResults = scope.Database.Fetch<BlogComment>().Find(x => x.BlogPostUmbracoId == umbracoNodeId);;
```

**Review by Zeegaan — APPROVED** [review]
LGTM 👍

**Comment by sofietoft** [comment]
Thanks for helping with a review @Zeegaan ! 🙌 

Looks like we can go ahead and merge this one in!

Thanks again for adding this to our docs @jannikanker ! 😄

**Pull request merged**